wdl-lint/tests/lints/nonmatching-output/source.wdl

```diff
@@ -0,0 +1,65 @@
+#@ except: DescriptionMissing, MissingRuntime
```

Comment:
We should test that:

- no diagnostics are output for an empty `output` section.
- what happens when the `meta.outputs` key is not an object.
- what happens when the user excepts the rule?

Reply:
I added exceptions with comments to indicate which work and which don't.